pod/perl.pod

```diff
@@ -171,6 +171,7 @@ aux h2ph h2xs perlbug pl2pm pod2html pod2man splain xsubpp
     perlhacktut 	Walk through the creation of a simple C code patch
     perlhacktips	Tips for Perl core C code hacking
     perlpolicy		Perl development policies
+    perlgov				Perl Rules of Governance
```

Comment:
This seems like an odd place (internals) but I guess it was placed next to perlpolicy? Either way the indentation is off.

Reply:
Correct, next to perlpolicy.  The indentation was my bad tabstop setting. :)